core/vtk/ttkPlanarGraphLayout/ttkMergeTreeVisualization.h

```diff
@@ -42,6 +42,15 @@ class ttkMergeTreeVisualization : public MergeTreeVisualization {
   // Offset
   int iSampleOffset = 0;
   int noSampleOffset = 0;
+  double prevXMaxOffset = 0;
+
+  // Print only one tree
+  int printTreeId = -1; // -1 for all
+  int printClusterId = -1; // -1 for all
+
+  // Barycenter position according alpha
+  bool BarycenterPositionAlpha = false;
+  double Alpha = 0.5;
```

Comment:
overall note:
the class `ttkMergeTreeVisualization.h` seems to be a somewhat unusual mixture of vtk and base code.
it is not a vtk filter per se (which makes it look like some base code) but it manipulates some vtk objects...

maybe all the functions which do not manipulate vtk objects should be moved to `core/base/planarLayout/MergeTreeVisualization.h`?
would that even make sense?

Reply:
Yes it makes total sense, however, the functions that are in `ttkMergeTreeVisualization.h` are all manipulating vtk objects.

Reply:
ok!